apps/vr-tests-react-components/src/stories/InfoButton.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { Steps, StoryWright } from 'storywright';
+import { InfoButton } from '@fluentui/react-infobutton';
+import { storiesOf } from '@storybook/react';
+import { TestWrapperDecoratorFixedWidth } from '../utilities/TestWrapperDecorator';
+
+storiesOf('InfoButton', module)
+  .addDecorator(TestWrapperDecoratorFixedWidth)
+  .addDecorator(story => (
+    <StoryWright
+      steps={new Steps()
+        .snapshot('rest', { cropTo: '.testWrapper' })
+        .hover('.info-button')
+        .snapshot('hover', { cropTo: '.testWrapper' })
+        // keys 'tab' is used instead of .focus, since .focus won't show the focus indicator.
+        .keys('.info-button', 'Tab')
+        .snapshot('focus', { cropTo: '.testWrapper' })
+        .click('.info-button')
+        .snapshot('active', { cropTo: '.testWrapper' })
```

Comment:
It looks like the popover is getting clipped in the active screenshots

Reply:
since we are testing the sizes, I think it's not needed, but I can make it fit.